### PR TITLE
Add accordion banners

### PR DESCRIPTION
**Pull request by balazsroman-tpx**
accordion banners added to enable jumping between accordions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "uk_gov_dash_components",
-  "version": "1.14.1",
+  "version": "1.15.1",
```
Comment (tom-clarke-dluhc) [inline package.json:3]: The version should go to 1.15.0 for a minor change

File: src/demo/App.js [hunk 2]
```diff
@@ -71,27 +71,27 @@ const App = () => {
...
                 <li>
                     <a>
                         Local authority map
                     </a>
                 </li>
             </ul>
-            <Accordion id="accordion" accordionHeadings={["Test", "HATS!!!!!"]} children={[<h1>I am a child</h1>,<h1>I am a hat</h1>]}></Accordion>
-            <Tabs id="tabs" tabHeadings={["Display jitter plots", "Display time series plots"]} defaultTab={0} children={[<div><h1>I am a jitter plot</h1></div>,<div><h1>I am a time series plot</h1></div>]}></Tabs>
+            <Accordion id="accordion" accordionHeadings={["charts", "data!!!!!"]} children={[<h1>I am a child</h1>, <h1>I am a hat</h1>]} links={[1, 0]}></Accordion>
+            <Tabs id="tabs" tabHeadings={["Display jitter plots", 'data', "Display time series pl}ots"]} defaultTab={0} children={[<div><h1>I am a jitter plot</h1></div>, <div><h1>I am a jitter plot</h1></div>, <div><h1>I am a time series plot</h1></div>]}></Tabs>
```
Comment (tom-clarke-dluhc) [inline src/demo/App.js:94]: Typo:
```suggestion
            <Tabs id="tabs" tabHeadings={["Display jitter plots", 'data', "Display time series plots"]} defaultTab={0} children={[<div><h1>I am a jitter plot</h1></div>, <div><h1>I am a jitter plot</h1></div>, <div><h1>I am a time series plot</h1></div>]}></Tabs>

```

File: src/lib/components/Accordion.react.js [hunk 3]
```diff
@@ -56,6 +56,12 @@ Accordion.propTypes = {
      * Array of accordion children.
      */
     children: PropTypes.arrayOf(PropTypes.node),
+
+    /**
+     * Indexes on elements to jump to.
+     */
+    links: PropTypes.arrayOf(PropTypes.number),
```
Comment (tom-clarke-dluhc) [inline src/lib/components/Accordion.react.js:63]: These names don't work together very well. Are they links? are they indices? Maybe more detail needed in docstring

File: src/lib/fragments/Accordion.react.js [hunk 4]
```diff
@@ -115,7 +116,19 @@ class Accordion extends Component {
     }
   }
 
-  findFocusableElement(element, direction, distance=1) { //distance is how many elements away the element you want to focus on is. 
+  jumpToAccordionContentSection(index) {
+    console.log('going to ', this.contentRefs[index])
```
Comment (tom-clarke-dluhc) [inline src/lib/fragments/Accordion.react.js:120]: do you still want this console.log?

File: src/lib/fragments/Accordion.react.js [hunk 5]
```diff
@@ -159,9 +178,18 @@ class Accordion extends Component {
     )
   }
 
-  renderAccordionSection(index, accordionSectionContent, sectionIsOpen) {
+  renderAccordionSection(index, accordionSectionContent, sectionIsOpen, link) {
     const accordionHeading = this.props.accordionHeadings[index]
+    const linkHeading = this.props.accordionHeadings[link]
     const contentId = `accordion-default-content-${index}`;
+    const styles = {
```
Comment (tom-clarke-dluhc) [inline src/lib/fragments/Accordion.react.js:185]: Is this the best place to store this CSS?

**Review by tom-clarke-dluhc — APPROVED** [review]
image
I had problems running but it doesn't look related to the PR. Otherwise looks good!

File: src/lib/components/Accordion.react.js (hunk 3 above)
Comment (urquha) [reply src/lib/components/Accordion.react.js:63]: good point

**Pull request merged**